src/main/java/savvy/core/entity/events/DoSelectFilterEntity.java

```diff
@@ -0,0 +1,9 @@
+package savvy.core.entity.events;
```

Comment:
UI events should be in their corresponding UI package
in this case it would be ui.entities_list
if more than a couple events are created there, put then in ui.entities_list.events

Event names for Dos should follow the pattern:
DoItemAction - in this case it would be DoEntityFilterSelect
Then a reactionary event would be called EntityFilterSelected
(of course, we only create the reactionary on if needed)